README.md

```diff
@@ -24,6 +24,12 @@
 npm install
 ```
 
+Install database
+
+```sh
+yay -S mariadb
+```
+
```

Comment:
Requires Arch, maybe link to this site: https://downloads.mariadb.org/mariadb/repositories/#distro=Ubuntu&distro_release=groovy--ubuntu_groovy&mirror=one&version=10.5

Reply:
Serves as an example